ml/main.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import FastAPI, File, UploadFile
+from pydantic import BaseModel
+from typing import List
+
+import files
+import paths
+import train as trainModule
+import utils
+
+app = FastAPI()
+
+
+class Encode(BaseModel):
+    text: List
+
+
+@app.get("/encode/{project}")
+def encode(encode: Encode, project):
+    return {"results": utils.encode(project, encode.text).tolist()}
+
+
+@app.post("/train/{project}")
+def train(project, file: UploadFile = File(...)):
```

Comment:
We'll need to add some guard rails around this function for the case where an invalid file is added.